framework/source/class/qx/ui/core/Widget.js

```diff
@@ -1360,9 +1360,10 @@ qx.Class.define("qx.ui.core.Widget",
       var contentWidth = width - insetX;
 
       // Compute height
+      var contentHeight =  0;
       var layout = this._getLayout();
       if (layout && layout.hasHeightForWidth()) {
-        var contentHeight =  layout.getHeightForWidth(contentWidth);
```

Comment:
It does not https://www.w3schools.com/js/js_hoisting.asp ... but I agree declaring it as you suggest looks less confusing